packages/flutter/lib/src/mountain_view/page.dart

```diff
@@ -0,0 +1,36 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/widgets.dart';
+
+/// A page route transition used for Android and Fuchsia to transition between pages.
+class MountainViewPageTransition extends AnimatedWidget {
```

Comment:
This seems to me like it's just a non-hero Material page transition. (You can tell because it uses fastOutSlowIn, which is a Material curve). I don't really see the point of having this in its own library.